front/app/components/IdeaCards/shared/Filters/TopicFilterBox.tsx

```diff
@@ -17,11 +17,15 @@ interface Props {
 
 const TopicFilterBox = memo<Props>(
   ({ selectedTopicIds, selectedIdeaFilters, onChange, className }) => {
-    const { data: topics } = useTopics();
+    const ideaFiltersWithoutTopics = omit(selectedIdeaFilters, 'topics');
+
+    const { data: topics } = useTopics({
+      sort: '-ideas_count',
+      ideas: ideaFiltersWithoutTopics,
+    });
 
     // Remove topics from selected idea filters so we get counts that aren't impacted by the selected topics.
     // Otherwise the counts change each time a topic is selected/deselected which isn't expected behaviour.
```

Comment:
Is this comment still relevant given the following line of code has been removed?

Reply:
Good catch! I'll remove it :+1: